client/src/routes/game/+page.svelte

```diff
@@ -64,6 +65,13 @@
       // will change privatePlayer to the new doc data
       privatePlayer = doc.data();
     });
+
+    addEventListener("beforeunload", (event) => {
+      event.returnValue = "\\o/";
+      if (lobby?.state === "WAIT" && lobbyCode !== null) {
```

Comment:
this code always runs even if the user clicks "cancel" maybe use the "unload" event

Reply:
We decided not to have the user leave during the unload event so this won't need to be addressed